gNMI_Interface/gnmi_config.py

```diff
@@ -0,0 +1,241 @@
+"""Utility classes for representation of gNMI device configuration."""
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from enum import Enum
+import typing as t
+import json
+
+from confd_gnmi_common import add_path_prefix, make_formatted_path
+
+import gnmi_pb2 as gnmi
+
+
+T = t.TypeVar('T', bound='UpdateIndex')
+
+
+class UpdateType(Enum):
```

Comment:
Describe (comment) in high level what these classes are for.

Reply:
Added.